app/frontend/src/components/base/BaseActionCard.vue

```diff
@@ -17,8 +24,15 @@ export default {
 </script>
 
 <style scoped>
-.v-card .action-card {
-  background-color: #003366;
-  border-top: 10px;
+.action-card {
+  height: 100%;
+}
+
+.v-system-bar {
+  background-color: #38598a;
```

Comment:
Maybe we should consider having this come into baseactioncard as a prop instead of embedded in here? Thoughts?

Reply:
Hmm yeah that's a good idea. Will leave that until next usage of BaseActionCard.